en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+  ```
+  10 + 20
+  ```
+- Concatenating two decimal literals
+  ```
+  0.56 + 1.84
+  ```
+- Concatenating a literal and a variable
+  ```
+  100 + nameVariable
+  ```
+- Concatenating two variables
+  ```
+  firstNumber + secondNumber
+  ```
+
+### Number Subtraction
+
+You can use the `-` operator for number subtraction. The number values can come from a literal or a variable.
+
+#### Examples
+- Subtracting two integer literals
+  ```
+  10 - 20
+  ```
+- Subtracting two decimal literals
+  ```
+  0.56 - 1.84
+  ```
+- Subtracting a literal and a variable
+  ```
+  100 - <named variable>
+  ```
+- Subtracting two variables
+  ```
+  <first number> - <second number>
+  ```
+
+### Number Division
+
+You can use the `/` operator for number division. The number values can come from a literal or a variable.
+
+#### Examples
+
+- Division between two integer literals
+  ```
+  10 / 20
+  ```
+- Division between two decimal literals
+  ```
+  0.56 / 1.84
+  ```
+- Division between a literal and a variable
+  ```
+  100 / nameVariable
+  ```
+- Division between two variables
+  ```
+  firstNumber - secondNumber
+  ```
+
+### Number Multiplication
+
+You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+
+#### Examples
+
+- Multiplying two integer literals
+  ```
+  10 * 20
+  ```
+- Multiplying two decimal literals
+  ```
+  0.56 * 1.84
+  ```
+- Multiplying a literal and a variable
+  ```
+  100 * nameVariable
+  ```
+- Multiplying two variables
+  ```
+  firstNumber * secondNumber
+  ```
+
+### Modulo Operation
+
+You can use the  `%` operator to get the remainder of a division. The number values can come from a literal or a variable.
+
+#### Examples
+- Modulus of two integer literals
+  ```
+  10 % 4
+  ```
+- Modulus of a literal and a variable
+  ```
+  100 % nameVariable
+  ```
+- Modulus of two variables
+  ```
+  firstNumber % secondNumber
+  ```
+
+### Checking The Equality of Basic Types 
+
+In the Ballerina language, basic types are strings, integers, decimal numbers, and booleans. You can compare the values of these types using the `==` binary operator. The values can come from a literal or a variable. Similarly `!=` operator is used to check the inequality. Please note that the resulting value is always of type boolean from these operators.
+
+#### Examples
+- Equality of two integer literals
+  ```
+  10 == 4
+  ```
+- Equality of two string literals
+  ```
+  "hi" == "hi"
+  ```
+- Equality of a literal and a variable
+  ```
+  100 == expectedValueVariable
+  ```
+- Equality of two variables
+  ```
+  firstNumber == secondNumber
+  ```
+- Inequality of two variables. The following will return true when the value of two variables are different.
+  ```
+  firstNumber != secondNumber
+  ```
+
+### Checking the Type of Variable
+
+The `is` operator is used to assert the type of a variable.
+
+#### Examples
+- Checking whether the variable is of type JSON
```

Comment:
It is better to say "`JSON` type" instead of "type JSON".

Reply:
done